javascript/javascript1/week1/task1.js

```diff
@@ -0,0 +1,7 @@
+console.log("Age-ify!") // This is Age-ify 
+
+let yearOfBirth = 1994;
+let yearFuture = 2034;
+let age = yearFuture - yearOfBirth;
```

Comment:
Hi Natalia!😀 Neat job here! Just remember that **const** is preferred over let and var when declaring your variables, so they can both be seen outside of their scope and maintain a constant value. If you need something more flexible, you can use **let**, but **var** is too flexible. You can read more about this right [here](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/).